metacat-functional-tests/src/functionalTest/groovy/com/netflix/metacat/MetacatSmokeSpec.groovy

```diff
@@ -318,6 +318,8 @@ class MetacatSmokeSpec extends Specification {
         createTable(catalogName, databaseName, tableName)
         api.deleteTable(catalogName, databaseName, tableName)
         def definitions = metadataApi.getDefinitionMetadataList(null, null, null, null, null, null, name,null)
+        createTable(catalogName, databaseName, tableName)
```

Comment:
We need a test to testing the partition deletion parts, it's better also testing the batching part if possible.